.github/workflows/stale_issues_and_pr_cleanup.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      DAYS_BEFORE_ISSUE_STALE: 180
+      DAYS_BEFORE_PR_STALE: 180
+      DAYS_BEFORE_CLOSE: 14
+    steps:
+      - uses: actions/stale@v7
```

Comment:
We had conversations around using 3rd party GitHub actions a long time ago (deciding not to), but it's probably a conversation worth revisiting the security of this as used here - seeing as there are multiple actions on our repo using 3rd party GitHub actions. https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Just an fyi, this is a GitHub authored action.

Reply:
I posted to security and they are comfortable with this approach.